definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -1690,5 +1690,17 @@
     "PostConditionState": "SUBMITTED_NOT_PAID",
     "SecurityClassification": "Public",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "01/03/2023",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "citizenCaseWithdraw",
+    "Name": "Withdraw Citizen Case",
+    "Description": "Withdraw a case submitted by citizen",
+    "PreConditionState(s)": "SUBMITTED_PAID;SUBMITTED_NOT_PAID;AWAITING_RESUBMISSION_TO_HMCTS;GATE_KEEPING;CASE_ISSUE",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/citizen-case-withdrawn-notification",
+    "PostConditionState": "CASE_WITHDRAWN",
```

Comment:
Should this always be withdrawn? even when the case is in ISSUED/ Gatekeeping state?

Reply:
Hi @AlokDatta, yes state will change to Withdrawn always as this event is only used in API when case is not issued yet(i.e in Submitted or Pending) state.

After the case is issued, withdrawal process is completely offline & this event is never invoked in those cases.
As you explained, withdrawal request status is always changed through an order to mark it as Closed.